docs/book/src/simple-external-plugin-tutorial/testdata/sampleexternalplugin/v1/cmd/flags.go

```diff
@@ -43,7 +43,13 @@ func flagsCmd(pr *external.PluginRequest) external.PluginResponse {
 	flagsToParse.Bool("api", false, "sets the api flag to true")
 	flagsToParse.Bool("webhook", false, "sets the webhook flag to true")
 
-	flagsToParse.Parse(pr.Args)
+	if err := flagsToParse.Parse(pr.Args); err != nil {
+		pluginResponse.Error = true
+		pluginResponse.ErrorMsgs = []string{
+			"failed to parse flags",
```

Comment:
would be better if we add error here?

example
```go
			"failed to parse flags: " + err.Error(),
```

Reply:
Hi @kersten ,

That seems great. But could you please help us understand why would not be possible or helpful to append in the error message the error faced?